src/main/resources/db/migration/V14__update_indexes.sql

```diff
@@ -0,0 +1,28 @@
+DROP INDEX idx_material_aid_offer_city;
+DROP INDEX idx_material_aid_offer_region;
+DROP INDEX idx_accommodation_offer_city;
+DROP INDEX idx_accommodation_offer_region;
+DROP INDEX idx_transport_offer_origin_city;
+DROP INDEX idx_transport_offer_origin_region;
+DROP INDEX idx_transport_offer_destination_city;
+DROP INDEX idx_transport_offer_destination_region;
+DROP INDEX idx_city_city;
+
+CREATE index idx_material_aid_offer_city ON material_aid_offer (lower(city));
+CREATE index idx_material_aid_offer_region ON material_aid_offer (lower(region));
+CREATE index idx_material_aid_offer_status ON material_aid_offer USING hash(status);
+CREATE index idx_material_aid_offer_user_id ON material_aid_offer (user_id);
```

Comment:
Maybe we can use hash indexes on `userId` columns as well?
According to this article: https://stackoverflow.com/questions/398884/postgresql-hash-index
it would have some advantage.

Reply:
I used the hash index. You're right